metashare/api/models.py

```diff
@@ -313,6 +326,20 @@ def notify_has_url(self):
         message = {"type": "SCRATCH_ORG_PROVISIONED", "payload": payload}
         async_to_sync(push.push_message_about_instance)(self, message)
 
+    def notify_refreshing_changes(self):
+        from .serializers import ScratchOrgSerializer
+
+        payload = ScratchOrgSerializer(self).data
+        message = {"type": "SCRATCH_ORG_UPDATED", "payload": payload}
+        async_to_sync(push.push_message_about_instance)(self, message)
```

Comment:
Every time I create a scratch org and receive a `SCRATCH_ORG_PROVISIONED` event, I'm also receiving a `SCRATCH_ORG_UPDATED` with the exact same org payload.

Reply:
Would you be OK with just the `SCRATCH_ORG_UPDATED` event, then? `PROVISIONED` doesn't have to be a distinct concept.

Reply:
It's not clear to me why this is happening -- what is causing `self.tracker.has_changed("currently_refreshing_changes")` to be truthy when the org is provisioned?

From my perspective, it's not ideal to combine these into one "updated" event, because we want to display a message to the user on successful org-creation but not every time the org model is updated.

Reply:
OK, I've been digging into this and I'm baffled; that isn't changed anywhere in the code as it stands. So I'm also not clear on why it's happening. I'll leave it as-is for now and try to step through the server's behavior.

Reply:
It would be nice to figure out, but also isn't actively breaking anything, so probably shouldn't hold up review/merge when this PR is ready.

Reply:
Seems like within the job a new `FieldTracker` is being instantiated, which means it loses track of which fields have already been saved (e.g. the "current" state). This causes new unnecessary events to be triggered on the `scratch_org.save()` call made from the job.

Added a task to https://trello.com/c/8uR5ZTuC.